src/main/groovy/com/athaydes/spockframework/report/vivid/VividAstInspector.groovy

```diff
@@ -177,10 +177,6 @@ class VividASTVisitor extends ClassCodeVisitorSupport {
                 }
 
                 codeCollector.add( statement )
-
-                if ( statement.statementLabel == 'where' ) {
-                    waitForNextBlock = true
```

Comment:
I wonder if this boolean can be removed now? This code was contributed by someone else, I don't remember why this was needed.

Reply:
Also, all of the tests still run green.

Reply:
I am a bit busy at the moment... will get to this in the next few days.